sql/mob_droplist.sql

```diff
@@ -16541,6 +16541,10 @@ INSERT INTO `mob_droplist` VALUES (3138,0,0,1000,1875,1000); -- (Nergal, Bata, A
 INSERT INTO `mob_droplist` VALUES (3139,0,0,1000,1875,1000); -- (Apollyon Demon, Taurus, Ahriman) ancient_beastcoin x 1-2
 INSERT INTO `mob_droplist` VALUES (3139,0,0,1000,1875,500);  -- (Apollyon Demon, Taurus, Ahriman) ancient_beastcoin x 1-2
 INSERT INTO `mob_droplist` VALUES (3140,0,0,1000,2826,350);
+INSERT INTO `mob_droplist` VALUES (3141,0,0,1000,16990,40);
```

Comment:
i think daihanya is more in line with 100 for rate
based on http://ffxidb.com/zones/153/ancient-goobbue
https://ffxiclopedia.fandom.com/wiki/Ancient_Goobbue
these are small sample sizes, however, but 40 seems lowish

Reply:
40 also isn't a valid TH0 bucket~! (50 would be if it were Rare, but I'm leaning towards Uncommon on this one due to the numbers you've posted)

Reply:
i didn't understand ffxidb, though i looked at it to base this off of.  TH0 = 10%, but TH2 = 8%?  I put 4% to make topaz's TH2 actually even out at around 10% like it says.  There is a report of someone going 0/27 with TH on it, comments seem to make it seem very low drop rate.

Reply:
Sometimes FFXIDB's different TH buckets will conflict with each other; I tend to lean towards going with whatever is the higher sample size. (Can see sample sizes on FFXIDB by hovering over the table cell).

In either case, now that TH's exact mechanics are known, new drop percentages should match a designated TH bucket (_which_ bucket is debatable). At the moment our TH code doesn't function like it should, but that'll need to be fixed at the code level, not the data level.

At some point we'll need to go through our existing data and "snap" it to the nearest TH bucket. This'll be a prerequisite _before_ we can properly fix the TH code - we can't implement bucket behavior if we don't have buckets!

Reply:
LIES! All lies I tell you! That katana is a myth! Doesn't exist!
*cries in terribad luck*

Reply:
would 50 suffice?

Reply:
Yeah, 50 would be fine